### PR TITLE
mod 5 homework

### PR DESCRIPTION
**Pull request by BenBagBag**

### CODE REVIEW

**Comment by jamdon** [comment]
Your package.json is missing a dependency meaning your server won't run without error for me after doing an npm install.

**Comment by BenBagBag** [comment]
Should be fixed now.

File: routes/truckRoutes.js [hunk 1]
```diff
@@ -1 +1,38 @@
...
+var router = express.Router();
+
+router.route('/')
+  .get(function(request, response) {
+    var foodTrucks = trucks.getTrucks();
+    response.send(foodTrucks);
+  })
+  .post(function(request, response) {
+    var newTruck = request.body;
+    if (newTruck) {
```
Comment (jamdon) [inline routes/truckRoutes.js]: Do you see the issue here?

File: trucks.js [hunk 2]
```diff
@@ -501,4 +503,81 @@ var foodTrucks = [
...
+}
+
+function removeTruck(name) {
+  var truckIndex = foodTrucks.findIndex(function (truck) {
+    return truck.name === name;
+  });
+  foodTrucks.splice(truckIndex, 1);
+}
+
+module.exports.addTruck = addTruck;
```
Comment (jamdon) [inline trucks.js]: I recommend using either `exports` or just 1 `module.exports`

**Comment by jamdon** [comment]
There's an issue in your POST route. I've left a comment inline.

**Comment by BenBagBag** [comment]
@jamdon fixed, I think.

**Comment by jamdon** [comment]
Works great! 👍

**Pull request closed without merging**